canvas.md

```diff
@@ -48,11 +49,19 @@
 * Development of a volatility prediction model with direct applications in investment and policy-making.
 
 ## Key Collaborators
-* Quantitative Investment Firms
-* Academic partners
-* Datasets
+* Dataset: Wind (high-frequency market data), FRED (macroeconomic indicators).
+* Industry Partners: Anonymous fund managers (strategy validation, real-world constraints).
 
 ## Methods
-* Data Integration
-* Exploratory Analysis
-* Correlation measures (Dynamic Time Warping)
+* Data Integration: Align mixed-frequency data (e.g., monthly GDP, daily stock returns) using DTW.
+Clean and normalize macro/financial datasets (Wind, Bloomberg, OECD).
+* Exploratory Analysis: Compute dynamic correlations (DCC-MIDAS) between EPU and sectoral returns.
+Identify regime transitions via HMMs and Bai-Perron structural break tests.
+* Modeling: Correlation measures (Dynamic Time Warping). Train LSTM networks on macro-time-series for volatility forecasting.
+Optimize portfolios using regime-dependent covariance matrices.
+
+## Road map
+* Finish fitting CPI,GDP before March 3rd.

```

Comment:
What do you mean by fitting CPI and GDP? Please describe -- perhaps in a github issue so you we can discuss if necessary